server/routes/barcode/FindRecipientView.ts

```diff
@@ -1,17 +1,30 @@
 import type { FindRecipientForm } from 'forms'
+import { Prison } from '../../services/prison/PrisonRegisterService'
 
 export default class FindRecipientView {
   constructor(
     private readonly findRecipientForm: FindRecipientForm,
     private readonly errors?: Array<Record<string, string>>,
-    private readonly barcode?: string
+    private readonly barcode?: string,
+    private readonly prisonRegister?: Array<Prison>
   ) {}
 
-  get renderArgs(): { form: FindRecipientForm; errors: Array<Record<string, string>>; barcode: string } {
+  get renderArgs(): {
+    form: FindRecipientForm
+    errors: Array<Record<string, string>>
+    barcode: string
+    prisonRegister: Array<Record<string, string>>
+  } {
     return {
       form: this.findRecipientForm,
       errors: this.errors || [],
       barcode: this.barcode,
+      prisonRegister: this.prisonRegister.map(prison => {
```

Comment:
Maybe sorting the prison array by prison name would be sensible thing to do here, so that the view gets its drop down option sorted nicely (currently they are presented in the order that the service returns them from the API)

I'll probably do this in the next PR ...